app/models/view/inventory.rb

```diff
@@ -5,9 +5,15 @@ module Types
 module View
   # A wrapper around event-driven InventoryAggregate for use in views.
   class Inventory
+    ItemDropdownOption = Struct.new(:id, :name)
```

Comment:
Would this clash with the OpenStruct PR?

Reply:
This PR was created before creating the generic `Option` class was introduced. Fixed.